src/main/java/org/sciborgs1155/robot/drive/StandardDriveMotor.java

```diff
@@ -0,0 +1,26 @@
+package org.sciborgs1155.robot.drive;
+
+import org.sciborgs1155.lib.constants.SparkUtils;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMax.IdleMode;
+
+public class StandardDriveMotor {
```

Comment:
I mentioned this previously, but this class should probably go. It's very creative, but unfortunately dealing with hardware can result in boilerplate sometimes.